apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/graphql/GraphQLDocumentParser.kt

```diff
@@ -371,6 +374,20 @@ class GraphQLDocumentParser(
         ?: emptyList()
   }
 
+  private fun GraphQLParser.SelectionSetContext?.inlineFragments(operationType: String): List<InlineFragment> {
```

Comment:
I think we want the result to be a `List<ParseResult<InlineFragment>>` to correctly track the used types. If you do something like:

```
query TestQuery {
  ... on Query {
    hero {
      appearsIn
    }
  }
}
```

`appearsIn` is an enum type that needs to be generated but won't be if the "usedTypes" are not "bubbled" up.

Reply:
Whoops, should be fixed by 0ed96fb!